contrib/devtools/copyright_header.py

```diff
@@ -41,7 +41,7 @@
 ]
 EXCLUDE_COMPILED = re.compile('|'.join([fnmatch.translate(m) for m in EXCLUDE]))
 
-INCLUDE = ['*.h', '*.cpp', '*.cc', '*.c', '*.py']
+INCLUDE = ['*.h', '*.cpp', '*.cc', '*.c', '*.mm', '*.py', '*.sh', '*.bash-completion']
```

Comment:
d596e3fd1f10a007f17a31e46256046cc7add567: this belongs to another PR (17052) + should also backport 14785 first I think (and add missing changes from 17398 on top of it)

Reply:
hmm... `*.mm` was added in 16314 actually